app/controllers/items_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ItemsController < ApplicationController
+  def show
+    @item = Item.find(params[:id])
```

Comment:
The US for this IC with Items was US 3-- for an Items Index page. So, we actually only needed an `index` action in this controller that could query `Item.all` to get all the items and display them on the index page.